llama-index-core/llama_index/core/retrievers/fusion_retriever.py

```diff
@@ -92,8 +92,9 @@ def _get_queries(self, original_query: str) -> List[str]:
         if self._verbose:
             queries_str = "\n".join(queries)
             print(f"Generated queries:\n{queries_str}")
+
         # The LLM often returns more queries than we asked for, so trim the list.
-        return response.text.split("\n")[: self.num_queries]
+        return [QueryBundle(q) for q in queries[: self.num_queries - 1]]
```

Comment:
Note the "- 1" here to match line #85. We should only be generating and using `self.num_queries - 1` based off the documentation. We should send the original query string in addition to the `self.num_queries - 1` queries.